# Service for sending command to a specific board
uint8 board_id
---
bool success
string message